Apps/WebClient/src/ClientApp/src/utility/errorTranslator.ts

```diff
@@ -0,0 +1,57 @@
+export default class ErrorTranslator {
+    public static getHumanReadable(errorCode: string): string {
```

Comment:
minor suggestion: ErrorTranslator.getHumanReadable --> ErrorTranslator.getDisplayMessage

Reply:
sure, I can change that.